experimental/origin-graphql/src/typeDefs/Growth.js

```diff
@@ -127,6 +128,8 @@ module.exports =
   type Mutation {
     invite(emails: [String!]!): InviteResponse
     enroll(campaignId: Int!): EnrollResponse
-    log(event: JSON!): LogResponse
+    gasForIdentity(walletAddress: ID!): SimpleResponse
```

Comment:
Spec is not finalized but I think we may give users enough ETH to pay for gas not only for Identity but also to create a listing. So maybe we give this mutation a more generic name like "sponsoredGas" ?

Reply:
Good point 👍